code/nuxeo-admin-center-jsf/src/main/resources/web/nuxeo.war/incl/connect/connectStatus.xhtml

```diff
@@ -234,19 +236,12 @@
           <h:outputText value="#{messages['label.nuxeo.connect.securityError']}" />
           <br />
           (<h:outputText value="#{connectStatus.status.errorMessage}" />)
-      </c:if>
+        </c:if>
 
         <c:if test="#{!connectStatus.status.isSecurityError()}">
           <h:outputText value="#{messages['label.nuxeo.connect.statusError']}" />
           <h:outputText value="#{connectStatus.status.errorMessage}" />
         </c:if>
-        <h:form>
-          <h:outputText value="#{messages['label.nuxeo.connect.unregister']}" />
-          <h:outputText value="&#160;" />
-          <a4j:commandButton value="#{messages['action.connect.unregister']}"
-            styleClass="button" action="#{connectStatus.unregister}"
-            render="connectStatusPanel" />
-        </h:form>
```

Comment:
Why removing the whole unregister part? If the status is on error you won't be able to unregister your instance to maybe fix the issue? That's right?

Reply:
if i'm not mistaken, the unregister is still there (it was duplicated in the initial page and display conditions have changed)

Reply:
Ok, right :)